src/odemis/acq/stream/_sync.py

```diff
@@ -851,22 +850,25 @@ def _adjustHardwareSettings(self):
             # Largest (square) resolution the dwell time permits
             rng = self._emitter.dwellTime.range
             pxs = self._getPixelSize()
-            max_tile_shape_dt = int(math.sqrt(exp / (rng[0] * 2)))
-            # Largest resolution the SEM scale permits (assuming min scale = 1)
-            sem_pxs = self._emitter.pixelSize.value
-            max_tile_shape_scale = int(pxs / sem_pxs[0])
-
-            # the min of both is the real maximum we can do
-            ts = max(1, min(max_tile_shape_dt, max_tile_shape_scale))
-            tile_shape = (ts, ts)
-            subpxs = pxs / ts
-            dt = (exp / numpy.prod(tile_shape)) / 2
-            scale = (subpxs / sem_pxs[0], subpxs / sem_pxs[1])
-
-            # Double check fuzzing would work (and make sense)
-            if ts == 1 or not (rng[0] <= dt <= rng[1]) or scale[0] < 1 or scale[1] < 1:
-                logging.info("Disabled fuzzing because SEM wouldn't support it")
+            if pxs[0] != pxs[1]:  # TODO: support fuzzing for rectangular pxs
```

Comment:
Please, in this case, just log a warning that it's not supported, and continue with the usual code. (I'm particularly worried with floating point errors).